src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -519,7 +519,29 @@ public Boolean call() throws InterruptedException {
         }
     }
 
-    /**
+  /**
+   * try to run the Java command in the PATH ad report its version.
+   * @param listener lister to print the output of the java command.
+   */
+  private void checkJavaIsInPath(TaskListener listener) {
+    String msg = "Java is not in the PATH nor configured with the javaPath setting,"
+                 + " Jenkins will try to guess where is Java,"
```

Comment:
The space after last coma is still missing here.

Reply:
```suggestion
                 + " Jenkins will try to guess where is Java, "
```